entrypoint.sh

```diff
@@ -93,7 +93,8 @@ if [ -t 0 ]; then
         {
             echo -e "\n${MARKER}"
             echo "if [ -f /usr/local/bin/activate ]; then"
-            echo "    source /usr/local/bin/activate"
+            echo "    DOCKER_CACHE_BASE_DIR=\"${PWD}\""
```

Comment:
Not for this line but there's also a `DOCKER_CACHE_DIR=${PWD}/.cache/docker` at the top of `entrypoint.sh`. Should that be updated as well?